src/element.ts

```diff
@@ -131,7 +138,7 @@ export abstract class Element {
   }
 
   getCategory(): string {
-    return (<typeof Element>this.constructor).CATEGORY;
+    return `${this.#attrs.type}`;
```

Comment:
Can we just return `this.#attrs.type` here? Isn't it already a string?

Reply:
done :)